.github/workflows/example-fix-commit.yaml

```
@@ -17,7 +17,7 @@ jobs:
     - name: Get changed files
       id: changed-files
       uses: tj-actions/changed-files@v35
-    - uses: kubescape/github-action@main
+    - uses: kubescape/github-action@feat-image-scanning
```

chore: once we implement proper versioning for our Github Action, this should refer to the tag

For example:
```suggestion
    - uses: kubescape/github-action@v1
```

Since I don’t think we would change the Github Action multiple times for the same Kubescape version  I would suggest using the same version tag as Kubescape. But that’s up to you.